Frameworks/UIKit/UIApplicationMain.mm

```diff
@@ -325,14 +329,24 @@ void _UIApplicationShutdown() {
     [[NSUserDefaults _standardUserDefaultsNoInitialize] _resumeSynchronize];
 }
 
-extern "C" void UIApplicationMainHandleToastNotificationEvent(const char* notificationData) {
-    NSString* data = Strings::IsEmpty<const char*>(notificationData) ? nil : [[NSString alloc] initWithCString:notificationData];
-    [[UIApplication sharedApplication] _sendNotificationReceivedEvent:data];
+extern "C" void UIApplicationMainHandleToastActionEvent(HSTRING toastArgument, IInspectable* toastUserInput) {
+    NSString* argument = Strings::WideToNSString(toastArgument);
+
+    // Convert to NSDictionary with NSStrings
+    NSMutableDictionary* userInput = [[NSMutableDictionary new] autorelease];
+    ComPtr<IInspectable> comPtr = toastUserInput;
+    ComPtr<ABI::Windows::Foundation::Collections::IPropertySet> map;
+    THROW_NS_IF_FAILED(comPtr.As(&map));
+    THROW_NS_IF_FAILED(Collections::WRLToNSCollection(map, &userInput));
```

Comment:
>userInput [](start = 60, length = 9)

based on what I read in the other one, this was callee allocated (hence being an outparam)?